backend/app/api/ws/device_namespace.py

```
@@ -931,6 +941,44 @@ async def _broadcast_device_status(
+
+            await sio.emit(
+                "device:slot_update",
+                event_data,
+                room=f"user:{user_id}",
+                namespace="/chat",
+            )
+            logger.debug(
+                f"[Device WS] Broadcast device:slot_update to user:{user_id}, "
+                f"slot_used={slot_info['used']}"
+            )
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)